### ISSUE TITLE
airflow-aws-executors version 1.1.2 - could not find this version

### ISSUE THREAD

**Issue by KristiKovacs** [aelzeiny/airflow-aws-executors#21]
Hello,

I noticed this issue when our requirements.txt with docker build. This is the error:

`Could not find a version that satisfies the requirement airflow-aws-executors==1.1.2 (from versions: 0.1.13, 0.1.14, 0.12, 1.0.0, 1.1.0, 1.1.2.post1)`

but I do se you have 1.1.2 in your setup.py https://github.com/aelzeiny/airflow-aws-executors/blob/1.1.2/setup.py#L15

**Comment by aelzeiny** [comment]
Let me double check to make sure that I didn’t break anything. Get back to you in a few minutes

**Comment by aelzeiny** [comment]
Fixing the issue

**Comment by KristiKovacs** [comment]
Thanks! Do you think it will be fixed today, or should I use another version in the meantime?

**Comment by aelzeiny** [comment]
So I goofed up quite a bit with the `1.1.2` release. Beginning with https://github.com/aelzeiny/airflow-aws-executors/issues/13. I tried to fix it, but PyPi doesn't allow the reuploading of versions (probably for good reason). Due to the confusion, I yanked `1.1.2`. Super-sorry, it won't happen again.

I went ahead and made a post-release for `1.1.2` called `1.1.2.post3` and an identical release called `1.1.3`. There are no notable changes between `1.1.2` and `1.1.3` that wouldn't be backwards compatible. So if you're reading this, please change your package version to `1.1.3`, and I humbly apologize for the inconvenience.

```
airflow-aws-executors (1.1.3)
Available versions: 1.1.3, 1.1.2.post3, 1.1.0, 1.0.0, 0.1.14, 0.1.13
  INSTALLED: 1.1.3
  LATEST:    1.1.3
```

**Comment by aelzeiny** [comment]
Closing the issue for now. Please reopen if you experience something else that's strange. Thank you for reporting this issue, I'll make an active effort to be more diligent with releases from here on out.
